rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] Make sure the rental is adding itself to the @book achieving the `belong-to` association.
- [ ] Make sure the rental is adding itself to the @person achieving the `belong-to` association.
Hint: use `self`